### ISSUE TITLE
Memory HCL: ERROR: Vendor BIOS HCL

### ISSUE THREAD

**Issue by Stanislaw-bnk** [Dasharo/dts-hw-conf-gen#3]
I managed to extract a `Memory` report for the Z690-A motherboard, which I committed [here](https://github.com/Dasharo/docs/pull/964/commits/120ee35e708ece905dbeaf1c15cd596433896ae1), using the `find . -name "Micro-Star_International_Co.,_Ltd._MS-7D25*.tar.gz" -print0 | xargs -0 -n1 bash -c './dts-hclmgr memory "$0"'`
However, the only thing I get for the Z790-P motherboard is:
```
ERROR: Vendor BIOS HCL
ERROR: Vendor BIOS HCL
ERROR: Vendor BIOS HCL
ERROR: Vendor BIOS HCL

gzip: stdin: unexpected end of file
tar: Child returned status 1
tar: Error is not recoverable: exiting now
ERROR: Vendor BIOS HCL
ERROR: Vendor BIOS HCL
ERROR: Vendor BIOS HCL
ERROR: Vendor BIOS HCL
ERROR: Vendor BIOS HCL
ERROR: Vendor BIOS HCL
```
So no results and lots of errors. Does this mean that there are no reports for this motherboard?

**Comment by pietrushnic** [comment]
It may be a bug, but I was generating those reports in the past. I will look into that when I find time.

**Comment by EduKav1813** [comment]
@Stanislaw-bnk This was worked on in #6 . The "ERROR: Vendor BIOS HCL" means that there is no dasharo version in the report. The script right now works in a way that if there is no Dasharo version, it aborts processing the report. I was able to generate a table from all HCL reports available, but it indeed might be the case that for `Z790-P` there are no reports with dasharo version.

**Comment by pietrushnic** [comment]
> The script right now works in a way that if there is no Dasharo version, it aborts processing the report.

Yes, and that is correct, because the whole HCL is about compatibility with Dasharo firmware, not proprietary firmware. We should not consider HCLs for proprietary firmware.

> I was able to generate a table from all HCL reports available, but it indeed might be the case that for Z790-P there are no reports with dasharo version.

I checked that, and it should not be the case. I have at least 2-3 confirmations that we should have such HCLs on our cloud, so something else has to be wrong.

**Comment by pietrushnic** [comment]
@Stanislaw-bnk can we assume based on https://github.com/Dasharo/dts-hw-conf-gen/pull/8#issuecomment-2834241295 that we can close this issue with PR #8 merge?
